src/sentry/static/sentry/app/views/eventsV2/tags.tsx

```diff
@@ -108,13 +108,18 @@ class Tags extends React.Component<Props, State> {
 
       return segment;
     });
-
+    // Ensure we don't show >100% if there's a slight mismatch between the facets
+    // endpoint and the totals endpoint
+    const maxTotalValues =
+      segments.length === 1
```

Comment:
Why only if there is 1 segment? Could we hit this scenario if there is more than one facet?

Reply:
I suppose that's possible. I'll change it.